package.json

```diff
@@ -7,5 +7,8 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "aditya",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "express": "^4.18.2"
```

Comment:
It's a good practice to lock down the version of your dependencies to avoid unexpected breaking changes in future updates. You can do this by removing the caret (^) before the version number. [medium]